lib/galaxy/web/stack/__init__.py

```diff
@@ -210,8 +210,8 @@ def get_app_kwds(cls, config_section, app_name=None):
             config_file = uwsgi_opt.get("galaxy_config_file")
         # check for --yaml or --json uWSGI config options next
         if config_file is None:
-            config_file = (UWSGIApplicationStack._get_config_file(uwsgi_opt.get("yaml"), yaml.safe_load, config_section)
-                           or UWSGIApplicationStack._get_config_file(uwsgi_opt.get("json"), json.load, config_section))
+            config_file = (UWSGIApplicationStack._get_config_file(uwsgi_opt.get("yaml"), yaml.safe_load, config_section) or
+                           UWSGIApplicationStack._get_config_file(uwsgi_opt.get("json"), json.load, config_section))
```

Comment:
We now ignore W503, see https://github.com/galaxyproject/galaxy/pull/2594

Reply:
So we need to update galaxy-lib then - I adjusted this to pass tests there. Thanks for the pointer.